src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -80,14 +85,24 @@
+      this.currentGeolocationPermissionGranted = permission.location === 'granted';
+      this.isDeviceLocationEnabled = true;
+    } catch(err) {
+      this.isDeviceLocationEnabled = false;
+      console.log(err)
+    }
   }
 
-  async askForCurrentLocationPermission() {
-    await Geolocation.requestPermissions();
-    await this.checkPermissionStatus();
+  async askForEnableLocationSettings() {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)